modules/ign.js

```diff
@@ -0,0 +1,220 @@
+    const months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sept", "Oct", "Nov", "Dec"];
+    username = `${months[bd.getMonth()]} ${bd.getDate()}`;
+  }
+  const users = await Module.db.ign.findUsername(username, system);
+  if (users.length == 0) return interaction.reply({ content: `No members have the IGN ${username} for ${foundSystem.name}.`, ephemeral: true });
+  const guild = interaction.guild;
+  const embed = u.embed().setTitle(`The following members have the IGN ${username} for ${foundSystem.name}`);
+  const wePlay = users.map(user => guild.members.cache.get(user.discordId))
+    .filter(usr => usr != null)
+    .sort((a, b) => a.displayName.toLowerCase().localeCompare(b.displayName.toLowerCase()));
+
+  const listChunks = [];
+  for (let i = 0; i < wePlay.length; i += PAGESIZE) {
+    listChunks.push(wePlay.slice(i, i + PAGESIZE));
+  }
+  embed.setDescription(listChunks[0].join('\n'));
+  interaction.reply({ embeds: [embed] });
+  if (listChunks.length > 1) {
+    listChunks.slice(1).forEach(chunk => {
+      interaction.followUp({ content: chunk.join("\n") });
+    });
+  }
+}
+
+const Module = new Augur.Module()
+.addInteractionCommand({
+  name: "ign",
+  guildId: sf.ldsg,
+  commandId: sf.commands.slashIgn,
+  process: async (interaction) => {
+    switch (interaction.options.getSubcommand(true)) {
+    case "view": return slashIgnView(interaction);
+    case "set": return slashIgnSet(interaction);
+    case "remove": return slashIgnRemove(interaction);
+    case "whoplays": return slashIgnWhoplays(interaction);
+    case "whois": return slashIgnWhois(interaction);
+    }
+  }
+})
+.setInit(async () => {
+  const doc = new GoogleSpreadsheet(config.google.sheets.config);
+  try {
+    await doc.useServiceAccountAuth(config.google.creds);
+    await doc.loadInfo();
+    const sheet = await doc.sheetsByTitle["IGN"].getRows();
+    const aliasSheet = await doc.sheetsByTitle["IGN Aliases"].getRows();
+    aliases = new u.Collection(aliasSheet.map(x => [x["Alias"]?.toLowerCase(), x["System"]?.toLowerCase()])).concat(new u.Collection(sheet.map(x => [x["Name"]?.toLowerCase(), x["System"]?.toLowerCase()])));
+    IGNs = new u.Collection(sheet.map(x => [x["System"]?.toLowerCase(), { system: x["System"]?.toLowerCase(), name: x["Name"], category: x["Category"] ?? "Game Platforms", link: x["Link"] }]));
+  } catch (e) { u.errorHandler(e, "Load IGN Systems"); }
+})
+.addEvent('interactionCreate', async interaction => {
+  if (interaction.type == "APPLICATION_COMMAND_AUTOCOMPLETE" && interaction.commandId == sf.commands.slashIgn) {
+    const focusedValue = interaction.options.getFocused()?.toLowerCase();
+    const filtered = IGNs.filter(choice => choice.name.toLowerCase().startsWith(focusedValue));
+    await interaction.respond(filtered.map(choice => ({ name: choice.name, value: choice.name })));
```

Comment:
Need to make sure we aren't sending a list of >25 values, or else it creates an error.